rootfs/watcher_entrypoint

```diff
@@ -30,6 +30,15 @@ importas -iu default_cookie default_cookie
 importas -D $default_cookie ERLANG_COOKIE ERLANG_COOKIE
 s6-env ERLANG_COOKIE=$ERLANG_COOKIE
 
+## NODE_DNS
+##
+## Used in clustering; by default wallet is configured to discover nodes via
+## DNS. Registering nodes to the cluster is a responsibility of cluster tools
+## e.g. Kubernetes.
+##
+importas -D localhost NODE_DNS NODE_DNS
+s6-env NODE_DNS=$NODE_DNS
```

Comment:
What's S6 being used for here? This is a different way from running services and I'd like to make sure we definitely need it.

Reply:
Basically it cleans up the process for you, so you don't have 15 bash process hang around because you logged into the container trying to debug something… if you `docker exec -it container erl -remsh "..."`, cleanup is required.
It also allows us to not the run the app as root (suid tools like s6-setuidgid).